backend/src/test/java/de/laura/project/api/service/TriviaApiServiceTest.java

```diff
@@ -0,0 +1,52 @@
+package de.laura.project.api.service;
+
+import de.laura.project.api.model.TriviaApiData;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class TriviaApiServiceTest {
+
+
+    @Test
+
+    public void getDataPointsFromApi() {
+
+        //GIVEN
+
+        String url = "https://opentdb.com/api.php?amount=10&category=17&difficulty=medium&type=multiple";
+
+        RestTemplate restTemplate = mock(RestTemplate.class);
+        String[] incorrectAnswers = {"incorrectAnswer", "wrongAnswer"};
```

Comment:
to initialize an array inline e.g. in a constructor you can also write: 
`new String[]{"value1", "value", "etc."}`